src/icons/microsoft-azure.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
please evaluate the SVG against the best practices guidelines!
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

i think we should remove the height/width and use `currentColor` instead of a white fill.